x-pack/metricbeat/module/mssql/db/data.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package db
+
+import (
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
+	schema = s.Schema{
+		"database": s.Object{
+			"id": c.Str("database_id"),
+		},
+		// Returns space usage information for the transaction log.
+		"log_space_usage": s.Object{
+			"total_bytes":             c.Int("total_log_size_in_bytes"),
+			"used_bytes":              c.Int("used_log_space_in_bytes"),
+			"used_percent":            c.Float("used_log_space_in_percent"),
+			"since_last_backup_bytes": c.Int("log_space_in_bytes_since_last_backup"),
```

Comment:
These fields could be
```
total.bytes
used.bytes
used.pct
since_last_backup.bytes
```
As in other modules.

Reply:
I did not know it, done :)

Reply:
Here you can find the conventions: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html

Reply:
Favorited :heart_eyes:

Reply:
I have just read [here](https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html) that we shouldn't use `.` in individual field names. Is it correct? What am I misunderstanding?

Reply:
This is correct. But it means in the above you need to use something like `s.Object{ ...` like you can see here in the dev guide: https://www.elastic.co/guide/en/beats/devguide/current/creating-metricsets.html Also check other metricsets for examples.